pymedphys/_experimental/cube.py

```diff
@@ -316,11 +313,7 @@ def contour_to_points(contours):
 
 
 def align_cube_to_structure(
-    structure_name: str,
-    dcm_struct: pydicom.dataset.FileDataset,
-    quiet=False,
-    niter=10,
-    x0=None,
+    structure_name: str, dcm_struct, quiet=False, niter=10, x0=None
```

Comment:
is there a way to not lose the typing of dcm_struct?

Reply:
There is a way. The function would need to be wrapped in a function factory. Essentially the function can't be defined at import time.

There might also be another way which is tidier. And that's to allow typing flags to not enact the optional import logic. That'd be a bit more work though. It's this approach that I was considering going down if more typing was to be used through the library. More thought here needed.

Reply:
For now removing it is the easiest way forward.

Reply:
ok by me.
losing the typing is not a huge issue, and the naming of the variable makes it relatively clear what type is expected